.github/workflows/azure.yml

```diff
@@ -11,8 +11,51 @@ defaults:
     shell: bash
```

Comment:
The reason this file appears as modified, and not as new, is because in order to test a workflow that is triggered on `workflow_dispatch`, the workflow needs to already be registered on the default branch.

Thus, I merged in two very simple PRs (#725, #727) to add the skeleton of the files.